packages/server/graphql/public/typeDefs/MeetingSeries.graphql

```diff
@@ -41,4 +41,9 @@ type MeetingSeries {
   When the meeting series was cancelled
   """
   cancelledAt: DateTime
+
+  """
+  The meetings that belong to the meeting series
+  """
+  activeMeetings: [NewMeeting!]!
```

Comment:
+1 Since recurrence is currently scoped only to team prompt meetings, we may need to keep this as TeamPromptMeeting for now.

Reply:
as now we have recurrence-related properties in a `NewMeeting` i think it's ok to leave it as is and avoid unnecessary casts. once MVP is done, I'd like to create a recurrence spec discussion task to move us forward